packages/ui/src/styles/resources/_css-variables.scss

```diff
@@ -33,7 +33,6 @@
   --va-info: var(--va-theme-blue);
   --va-success: var(--va-vue-green);
   --va-secondary: #babfc2;
-  --va-light-gray2: #eff4f5;
```

Comment:
it's a duplicate from https://github.com/epicmaxco/vuestic-ui/blob/f5995bf867112c1f81cef16a45c8a66cff58d7a4/packages/ui/src/styles/resources/_css-variables.scss#L5